src/main/java/com/dreamhorizon/enchantments/enchantments/Poison.java

```diff
@@ -25,34 +25,38 @@
 import org.bukkit.NamespacedKey;
 import org.bukkit.enchantments.EnchantmentTarget;
 import org.bukkit.entity.LivingEntity;
+import org.bukkit.inventory.ItemStack;
```

Comment:
What was the purpose of this change? Do you like to keep a certain order?

Reply:
alphabetical with direct classes infront